app/views/govuk_publishing_components/components/_tabs.html.erb

```diff
@@ -1,5 +1,9 @@
 <%
   tabs ||= []
+  panel_border ||= true unless panel_border == false
```

Comment:
I don't think you need the "unless" clause. 
If you want the border false, you pass in panel_border = false and it takes that, the ||= just means "if nothing is passed in, set this value.

Reply:
"if nothing is passed in" is the same with false since the boolean operator is looking for any falsy value

Reply:
Ah, right. Ok as is then.